addon/addon-main.js

```diff
@@ -0,0 +1,9 @@
+const { addonV1Shim } = require('@embroider/addon-shim');
+module.exports = {
+  ...addonV1Shim(__dirname),
+  init(...args) {
+    this._super.init.apply(this, args);
+
+    this.treePaths.addon = 'src';
```

Comment:
this is needed for `ember-cli-addon-docs` to property find addon code, part of https://github.com/ember-learn/ember-cli-addon-docs/pull/1164

I was thinking we actually should move this to `@embroider/addon-shim`, @ef4 what do you think?